packages/vision/src/helpers.js

```diff
@@ -74,11 +74,12 @@ var coerceImage = (image, callback) => {
  *   asking for the single feature annotation.
  */
 var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
+  return function(image, options = {}, callOptions = undefined) {
```

Comment:
The way the code is in this PR, the API isn't working as the docs advertise. What we can do is find a different way to handle these concepts. JavaScript idioms have always preferred a configuration object over multiple arguments. We can use this to label the parts of the users intent:

```js
method(shortHandArgument, [options])
```

In this case:

```js
cropHints({ image }, {
  request: { imageContext: {} },
  callOptions: {...}
})
```

Or we can combine the `request` with the `callOptions` in a slightly different way:
```js
cropHints({ image }, {
  imageContext: {},

  // this is a whitelist property that is plucked from an options object and treated as `callOptions`
  gaxOptions: {}
})
```

If you want to keep the concept the way it is in this PR, you would have to remove the `optional` part of the arguments, and require users pass in undefined/null:

```js
cropHints({image}, null, {callOptions})
```